docs/source/tables/quality_header.csv

```diff
@@ -23,15 +23,19 @@ Name,Description,Example,Conditional requirement conditions
-*#^X_Chromatic_Shift:*,"This field captures the offset in the observed Y-coordinate of the Intensity maxima or the Intensity centre of gravity of the bright Spot when comparing the Reference (_R) vs. the Test (_T) wavelengths. This shall be calculates as: √(Y_T - Y_R)^2. This offset could be reported either in number of Pixels or in physical Distance, when a sub-Pixel offset needs to be calculated.",,Conditional requirement: this column name should be used if this metric is reported.
-*#^Y_Chromatic_Shift:*,"This field captures the offset in the observed Y-coordinate of the Intensity maxima or the Intensity centre of gravity of the bright Spot when comparing the Test (_T) vs. the Reference (_R) wavelengths. This shall be calculates as: √(Y_R - Y_T)^2. This offset could be reported either in number of Pixels or in physical Distance, when a sub-Pixel offset needs to be calculated.",,Conditional requirement: this column name should be used if this metric is reported.
-*#^Z_Chromatic_Shift:*,"This field captures the offset in the observed Z-coordinate of the Intensity maxima or the Intensity centre of gravity of the bright Spot when comparing the Test (_T) vs. the Reference (_R) wavelengths. This shall be calculates as: √(Z_T - Z_R)^2. This offset could be reported either in number of Pixels or in physical Distance, when a sub-Pixel offset needs to be calculated.",,Conditional requirement: this column name should be used if this metric is reported.
-*#^X_Loc_Error:*,"Empirically calculated error (i.e., uncertainty) associated with the estimation of the X-axis localization of this bright Spot. A description of how this uncertainty was computed, must be provided in the header. Such description must contain enough details to allow interpretation and reproducibility.",,Conditional requirement: this column name should be used if this metric is reported.
-*#^Y_Loc_Error:*,"Empirically calculated error (i.e., uncertainty) associated with the estimation of the X-axis localization of this bright Spot. A description of how this uncertainty was computed, must be provided in the header. Such description must contain enough details to allow interpretation and reproducibility.",,Conditional requirement: this column name should be used if this metric is reported.
+*#^X_Chromatic_Shift:*,"This field captures the offset in the observed X-coordinate of the Intensity maxima or the Intensity centre of gravity of the bright Spot when comparing the Reference (_R) vs. the Test (_T) wavelengths. This shall be calculates as: √(X_T - X_R)^2. This offset could be reported either in number of Pixels or in physical Distance, when a sub-Pixel offset needs to be calculated.",,Conditional requirement: this column name should be used if this metric is reported.

```

Comment:
Please double-check the formula and the order of items in the subtraction. This was written in different ways for X, Y and Z. I made them consistent, but want to make sure that my correction makes sense.

Reply:
Thank you for picking this up. It is now fixed.